### PR TITLE
Fix routing aa1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the routing is broken. Pages like `/contribute` can not be accessed on its own. (404 GitHub pages)

**What is the new behavior (if this is a feature change)?**
- The routing has been fixed.

**Other information**:
- The version has been updated to `1.5.2`.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -126,6 +126,43 @@
...
+      // This script checks to see if a redirect is present in the query string
+      // and converts it back into the correct url and adds it to the
+      // browser's history using window.history.replaceState(...),
+      // which won't cause the browser to attempt to load the new url.
+      // When the single page app is loaded further down in this file,
+      // the correct url will be waiting in the browser's history for
+      // the single page app to route accordingly.
+      (function (l) {
+        if (l.search) {
+          var q = {};
```
Comment (Theralun) [inline public/index.html:144]: Is var necessary here?

File: public/index.html [hunk 2]
```diff
@@ -126,6 +126,43 @@
...
+          l.search
+            .slice(1)
+            .split("&")
+            .forEach(function (v) {
+              var a = v.split("=");
```
Comment (Theralun) [inline public/index.html:149]: Is var necessary here?

File: public/index.html (hunk 1 above)
Comment (Aichnerc) [reply public/index.html]: The varcrime has been fixed.

File: public/index.html (hunk 2 above)
Comment (Aichnerc) [reply public/index.html]: The varcrime has been fixed.

**Pull request merged**